installer/.gitignore

```diff
@@ -0,0 +1,32 @@
+etc/bootstrap-local.conf
+
+.class
+*.log
+.history
+
+target/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+project/activator-sbt*
+
+.env
+
+# IntelliJ
+.idea/
+*.iml
+*.iws
+
+# Metals
+.metals/
+.bloop/
+
+# Mac
+.DS_Store
+
+# vim swap files
+*.swp
+.*.swp
+
+.vscode
```

Comment:
@maasg If u decide to copy this at the root level, please do not overwrite the one there. I have some entries there that are required for the release scripts to work.